content/pages/volunteers.md

```diff
@@ -16,16 +16,16 @@ We have a range of volunteer positions, each of which will help you to learn
 about an aspect of our committee. By volunteering to be a **photographer**, you
 can learn about how we run events and help with publicizing them. By
 volunteering to be a **technical mentor** for our events, you can help teach
-technical concepts to others. By becoming a **systems committee volunteer**, you
-can learn systems administration and static website development from our systems
-committee.
+technical concepts to others. We will no longer be accepting volunteers
```

Comment:
mmm sounds a little blunt, and not enough context I think?

"We currently don't have capacity to support volunteers for our systems committee, and will not be accepting systems committee volunteers until further notice."

(something like that, doesn't have to be that)